.travis.yml

```diff
@@ -16,7 +16,7 @@ install:
     - npm install
     - npm link
     - popd
-    - git clone --branch=release/4.9.x.develop --depth 1 https://github.com/enactjs/enact ../enact
+    - git clone --branch=feature/WRR-11077-4.9 --depth 1 https://github.com/enactjs/enact ../enact
```

Comment:
Don't forget to revert the change before merging.

Reply:
It's fixed.